src/pages/rest/b2b/company.md

```diff
@@ -0,0 +1,16 @@
+---
+title: Integrate with the Company module
+description: Overview of the Company module
+ee_only: True
+---
+
+# Integrate with the Company Module
+
+The `Company` module allows multiple buyers that belong to the same company to view product prices and make purchases based on the shared catalog assigned to this company. A company can contain a hierarchy of users and teams, with roles and permissions assigned at any level.
+
+## Related information
```

Comment:
I recommend deleting any "related information" section that reproduces the left nav.

Reply:
Yes, I had deleted them all, but several overview pages like this one looked very sparse. So I was wondering if it is better to leave them or somehow modify the page. Otherwise this is just a page with 2 sentences.

Reply:
Yeah, I've encountered similar dilemmas in other repos. As long as we don't make a habit of it, I think it's fine.